Tests/test_SeqFeature.py

```diff
@@ -28,13 +28,26 @@ def test_eq_identical(self):
         rec1 = SeqIO.read(testfile, "genbank")
         rec2 = SeqIO.read(testfile, "genbank")
 
-        self.assertEqual(rec1.annotations["references"][0], rec1.annotations["references"][0])
-        cmp1, cmp2 = rec1.annotations["references"][0], rec2.annotations["references"][0]
+        self.assertEqual(
+            rec1.annotations["references"][0], rec1.annotations["references"][0]
+        )
+        cmp1, cmp2 = (
```

Comment:
I don't see why ``cmp1`` and ``cmp2`` were defined, this looks exactly the same as the test above.

If you agree, we might as well remove the duplication.

Reply:
is slightly different, I have rewritten it so is more clear what each assert is doing.

Reply:
That's much clearer now 👍